<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DaqSet</class>
 <widget class="QWidget" name="DaqSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>选择读取通道数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_pass">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>连接类型:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_485">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;如果是moxa卡，这个按钮禁用。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>485</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_net">
          <property name="text">
           <string>网口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>数采卡型号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comB_cardType">
          <item>
           <property name="text">
            <string>1211</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1240</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4510</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>思迈科华</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>IP地址</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="le_ip1">
              <property name="maxLength">
               <number>3</number>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">font: 24pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_ip2">
              <property name="maxLength">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="styleSheet">
               <string notr="true">font: 24pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_ip3">
              <property name="maxLength">
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="styleSheet">
               <string notr="true">font: 24pt &quot;Arial&quot;;</string>
              </property>
              <property name="text">
               <string>.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_ip4">
              <property name="maxLength">
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_saveInfo">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>设备地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_deviceAddr"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>时间间隔</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_timeInterval"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>数采卡类型:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_Moxa">
          <property name="text">
           <string>MOXA</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_smacq">
          <property name="text">
           <string>思迈科华</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>先选择表格中某行数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_start">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_stop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="table_Msg">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MOXA型号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>时间间隔</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总时长(秒)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总次数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>失败次数</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="te_showMsg">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>收发信息</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
